plugins/templates/create.js

```diff
@@ -26,18 +29,22 @@ module.exports = () => ({
             const templateFactory = request.server.app.templateFactory;
             const pipelineId = request.auth.credentials.pipelineId;
             const name = request.payload.name;
-            const version = request.payload.version;
             const labels = request.payload.labels || [];
 
             return Promise.all([
                 pipelineFactory.get(pipelineId),
                 templateFactory.list({ name })
             ]).then(([pipeline, templates]) => {
+                const VERSION_REGEX = schema.config.regex.VERSION;
+                const match = VERSION_REGEX.exec(request.payload.version);
+                const major = match[MAJOR_MATCH];
+                const minor = match[MINOR_MATCH];
+                const version = minor ? `${major}${minor}.0` : `${major}.0.0`;
```

Comment:
second group includes the `.`: https://github.com/screwdriver-cd/data-schema/blob/master/config/regex.js#L11